calc_app/utils.py

```diff
@@ -0,0 +1,60 @@
+import matplotlib.pyplot as plt
+import base64
+from io import BytesIO
+
+
+def get_month_name(month_number):
+    dct = {
```

Comment:
Could u represent such values as table with constant rows?

Reply:
I think that it is possible, but I need to think of on how to implement it in the best way

Reply:
> `dct` - bad naming

Updated